README.md

```diff
@@ -25,14 +25,14 @@ We are moving development of Fractal v2.0 and all first-party add-ons into a 'mo
 #### Running locally
 
 1. Clone this repository
-2. Install dependencies - `npm install`
-3. Bootstrap the packages together using Lerna - `npm run bootstrap`
+2. Install dependencies - `yarn install`
```

Comment:
Ok, I'd thought that might be the case, but it felt a bit weird, having read the Readme recommending yarn, and then running the scripts which were all yarn, but then having the npm ref - it looked a bit like an oversight. 
Maybe we could just update the readme to be more explicit about that?